tests/test_torque_driven_with_contact_ocp.py

```diff
@@ -108,6 +111,9 @@ def test_maximize_predicted_height_CoM_with_actuators(ode_solver):
     # save and load
     TestUtils.save_and_load(sol, ocp, False)
 
+    # simulate
+    # TestUtils.simulate(sol, ocp)
```

Comment:
Same here

Reply:
Actually, this should not be commented, but it should be a negative test (one that fails with specific raise). So when we allow for that, this test will stop working so we will remember to add it back

Reply:
Are you okay with commit dc2734 ?

Reply:
Nope, but with the later commit yes ;)